modules/installation-and-upgrade/pages/client-requirements.adoc

```diff
@@ -189,7 +189,7 @@ ifeval::[{uyuni-content} == true]
 | {check}
 
 | {almalinux} 9
-| {x86_64}, {aarch64}
+| {x86_64}, {aarch64} {ppc64le}, {ibmz}, {s390x}
```

Comment:
What's the  difference between `{ibmz}` and `{s390x}`? You have both here, but you don't have both at `modules/client-configuration/pages/supported-features.adoc` (in fact you replaced `{ibmz}` there)